src/wymeditor/editor/image-handler.js

```diff
@@ -424,10 +424,15 @@ WYMeditor.ImageHandler.prototype._startResize = function (startMouseY) {
 WYMeditor.ImageHandler.prototype._onMousemove = function (evt) {
     var ih = this;
 
+    // default action for this event may be a `mousedown` and `mousemove`
+    // with the intention to select text
+    // or to drag (for later dropping of) content
+    // so be careful about preventing default
```

Comment:
This is the culprit.

Now it returns false only when it is surely not a text selection attempt.